src/components/iptables/README.md

```diff
@@ -18,7 +18,7 @@ make && sudo make install
 
 ```
 # Start polycubed, in other terminal (or background)
-sudo polycubed
+sudo polycubed &
```

Comment:
Another possibility is to launch polycubed in daemon mode.

Reply:
This is in fact the best way to do it.
To use polycubed --daemon.

Reply:
Wasn't the `d` at the end of polycube resembles `daemon`. I don't know what that `d` is then :)
Certainly, I'll modify this option with the neat option `--daemon`.

Reply:
@mauriciovasquezbernal I agree with Aasif, I had the same issue in the past. Having 'd' at the end of 'polycubed', I also assumed it runs as a daemon. Personally, I feel more clear to reverse the behaviour, i.e., polycubed runs as daemon, while something else like 'polycubed -foreground' runs in the terminal.